ts/sagas/contentLoaders.ts

```diff
@@ -43,7 +49,7 @@ export function* watchContentServiceLoadSaga(): Iterator<Effect> {
       serviceId
     );
 
-    if (response.isRight() && response.value.status === 200) {
```

Comment:
a service is loaded successfully if the response is right and the value's status is 200, so removing the second condition is an error (the response could be right and its status could be 404 for example)